x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -29,6 +29,36 @@ export const removeServerGeneratedPropertiesFromConfigure = (
   return rest;
 };
 
+export const deleteCasesUserActions = async (es: any): Promise<void> => {
+  await es.deleteByQuery({
+    index: '.kibana',
+    q: 'type:cases-user-actions',
+    waitForCompletion: true,
```

Comment:
If you consider switching to new `es` client you should change the arguments to:

```
wait_for_completion: true,
refresh: true,
body: {},
```